source/Purchases.brs

```diff
@@ -783,7 +789,9 @@ function _InternalPurchases(o = {} as object) as object
             offerings = result.data
             current_offering_id = offerings.current_offering_id
             current_offering = invalid
-            all_offerings = []
+            placements = offerings.placements
+            targeting = offerings.targeting
+            all_offerings = {}
```

Comment:
Fixing a mistake with `offerings.all`: It should be a dictionary indexed by offering id, but I implemented it as a simple array of all offerings.